pkg/operator/controller/nodeobservability/controller.go

```diff
@@ -381,3 +396,21 @@ func isClusterNodeObservability(ctx context.Context, nodeObs *operatorv1alpha2.N
 
 	return fmt.Errorf("a single NodeObservability with name 'cluster' is authorized. Resource %s will be ignored", nodeObs.Name)
 }
+
+// RequeuePeriod returns requeue period as time.Duration.
+func RequeuePeriod(RequeuePeriodSeconds int) time.Duration {
+	return time.Duration(RequeuePeriodSeconds) * time.Second
+}
```

Comment:
This can be replaced by 1 constant I mentioned in the suggestion above: `defaultRequeuePeriod`.